web/viewer.css

```diff
@@ -903,12 +871,11 @@ html[dir="rtl"] #toolbarViewerLeft > .toolbarButton:first-child {
   margin-top: 2px !important;
 }
 .dropdownToolbarButton::after {
-  position: absolute;
-  display: inline-block;
   top: 6px;
-  content: var(--toolbarButton-menuArrow-icon);
   pointer-events: none;
-  max-width: 16px;
+
+  -webkit-mask-image: var(--toolbarButton-menuArrow-icon);
```

Comment:
Doesn't Autoprefixer handle this for us so we don't have to worry about vendor prefixes anymore (same below)?

Reply:
Annoyingly enough, Chromium doesn't support the *un*prefixed version of this CSS property and I thus opted to include this to make sure that the *development* viewer doesn't look completely broken locally in Chromium-browsers.

I should obviously have mentioned, sorry about that!